src/blocks/CardGrid/index.module.scss

```diff
@@ -14,37 +15,45 @@
     }
 }
 
-.richText {
-    margin-bottom: calc(var(--base) * 3);
-    @include mid-break {
-        margin-bottom: var(--base);
-    }
+.richTextWrapper {
+  grid-area: 1 / 1 / 1 / -1;
+  align-items: center;
+  justify-content: center;
+  display: grid;
+
+  @include mid-break {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: flex-start;
+  }
 }
 
 .bg {
-    position: absolute;
-    top: 0;
-    left: 0;
-    width: calc(100% + var(--gutter-h));
-    height: 100%;
-    margin-left: calc(var(--gutter-h) / -2);
-
-    @include mid-break {
-        margin-left: calc(var(--gutter-h) * -1);
-        width: calc(100% + (var(--gutter-h) * 2));
-    }
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: calc(100% + var(--gutter-h) * 2);
+  height: 100%;
+  margin-left: calc(var(--gutter-h) / -1);
 }
 
 .cards {
-    position: relative;
-    padding: calc(var(--base) * 4) 0;
-
-    @include mid-break {
-        padding: calc(var(--base) * 2) 0;
-    }
+  position: relative;
 }
 
 .card {
     height: 100%;
     position: relative;
-}
\ No newline at end of file
+}
```

Comment:
@paulpopus According to the Figma mockup, the leader (aka the numbers in the top left corner) of the `SquareCards` are a different color and size.

So these will need some tweaking. Looks like the SquareCards component is only utilized for the CardGrid so the edit will most likely need to happen in that component. Here are some images below of what I'm referring to:

Current:
![Screen Shot 2024-01-21 at 12 41 16 PM](https://github.com/payloadcms/website/assets/35232443/fd005019-7f04-4167-b093-2a28ae72d211)

Mockup:
![Screen Shot 2024-01-21 at 12 42 10 PM](https://github.com/payloadcms/website/assets/35232443/640ffe47-3d4d-4da7-bb9a-1ec6d8bb84bf)